Cubical/Foundations/Bundle.agda

```diff
@@ -0,0 +1,55 @@
+{-# OPTIONS --cubical --safe #-}
+module Cubical.Foundations.Bundle where
```

Comment:
I'm starting to wonder what really belongs in Foundations... Could this go somewhere else, maybe in a new directory? I would like to move a lot of the content from Foundations just to keep it minimal. I think files should only be in Foundations if they are used in almost all other files in the library

Reply:
I very much agree! I don't know what this other directory should be called, though. `Cubical.Theories`? If we want to make a less drastic change we could make a `Cubical.Maps` or `Cubical.Functions` directory and just move `Bundle`, `TotalFiber`, `Surjection`, and `Embedding`, but I think a directory like you're describing will have to be made eventually.

Reply:
Cubical.Functions sounds pretty good for the things you mention. There might be some other things that could go there, e.g. FunExt.

Make an issue and fix in a future PR?